src/container/srv_layout.h

```diff
@@ -48,6 +48,7 @@ extern d_iov_t ds_cont_prop_version;		/* uint32_t */
 extern d_iov_t ds_cont_prop_cuuids;		/* container UUIDs KVS */
 extern d_iov_t ds_cont_prop_conts;		/* container KVS */
 extern d_iov_t ds_cont_prop_cont_handles;	/* container handle KVS */
+extern d_iov_t ds_cont_prop_global_version;	/* uint32 */
```

Comment:
Note that this is per-pool, not per-container. If the intention is to have a per-container property, we need to add to the Container Property KVS below. If the intention is to have a per-pool property, we can just export ds_pool_prop_global_version via some ds_pool API or field, without any new keys here.